test/test_jit_fuser.py

```diff
@@ -715,6 +716,7 @@ def test_lstm_cuda(self):
                                                   "aten::_grad_sum_to_size"))
 
     @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
+    @with_tf32_off
```

Comment:
I recommend comments explaining why tf32 is disabled for this test and test_lstm_traced_cudua, too.

Reply:
fixed